local.yml

```diff
@@ -2,5 +2,11 @@
 - hosts: localhost
   connection: local
   become: yes
+  vars:
+    ansible_python_interpreter: "{{ ansible_playbook_python }}"
```

Comment:
Do you remember why `ansible_python_interpreter` was added here?  It's set automatically by ansible exactly as you have done here with all versions of ansible back to 2.4.  (2.4 does not template ansible_x_interpreter values, so this playbook fails on 2.4.)